homeassistant/components/todoist/calendar.py

```diff
@@ -330,7 +330,7 @@ def __init__(
         # The latest date a task can be due (for making lists of everything
         # due today, or everything due in the next week, for example).
         if latest_task_due_date is not None:
-            self._latest_due_date = dt.utcnow() + timedelta(days=latest_task_due_date)
+            self._latest_due_date = timedelta(days=latest_task_due_date)
```

Comment:
This used to be an absolute date, and now it is a relative time delta, so I think the variable name needs to change as 
"_latest_due_date" implies that it is a date.  Maybe "due_date_days" like the config to keep it simple?